src/Security/Authentication/OpenIdConnect/src/OpenIdConnectHandler.cs

```diff
@@ -34,7 +34,7 @@ public class OpenIdConnectHandler : RemoteAuthenticationHandler<OpenIdConnectOpt
         private const string NonceProperty = "N";
         private const string HeaderValueEpocDate = "Thu, 01 Jan 1970 00:00:00 GMT";
 
-        private OpenIdConnectConfiguration _configuration;
+        private OpenIdConnectConfiguration? _configuration;
```

Comment:
Warning: This file is a bit of a garbage fire when it comes to nullability. Methods are BIG and reuse variables a lot.

Lots of `!`